appeals/web/src/server/appeals/appeal-details/appellant-case/address/address.validator.js

```diff
@@ -1,11 +1,19 @@
-import { createPostcodeValidator } from '#lib/validators/address.validator.js';
+import {
+	createAddressLine1Validator,
+	createTownValidator,
+	createPostcodeValidator
+} from '#lib/validators/address.validator.js';
+import { createTextInputOptionalValidator } from '#lib/validators/text-input-validator.js';
 import { createValidator } from '@pins/express';
-import { body } from 'express-validator';
 
 export const validateChangeSiteAddress = createValidator(
-	body('addressLine1').trim().notEmpty().withMessage('Enter the first line of the address'),
-	body('town').trim().notEmpty().withMessage('Enter the town'),
+	createAddressLine1Validator(),
+	createTextInputOptionalValidator(
+		'addressLine2',
+		250,
+		'Address line 2 must be 250 characters or less'
```

Comment:
Should this use a constant for the error message?

Reply:
Also might be worth storing the character limit value in a constant, similar to the procedure-preference validator

Reply:
Error messages are consistently static within validators on the web. I thought of refactoring the 'create' validators to dynamically build out error messages as they are very consistent, but I drew the refactor line there.
I've pulled in the max length as a constant though.